.github/workflows/ci.yml

```diff
@@ -52,6 +55,7 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.PYTHON.VERSION }}
+          allow-prereleases: true
```

Comment:
On our other repos we've preferred not to use `allow-prereleases`, and instead to explicitly use `3.12-dev` until it's released.

Reply:
Done!